src/uct/api/uct.h

```diff
@@ -904,8 +904,9 @@ struct uct_md_attr {
         size_t               max_alloc; /**< Maximal allocation size */
         size_t               max_reg;   /**< Maximal registration size */
         uint64_t             flags;     /**< UCT_MD_FLAG_xx */
-        uint64_t             reg_mem_types; /** UCS_BIT(uct_memory_type_t) */
-        uct_memory_type_t    mem_type;  /**< Supported(owned) memory type */
+        uint64_t             reg_mem_types; /**< Memory types MD can be registered with */
```

Comment:
I don't think we define MD acronym anywhere in the spec. It is more readable to use Memory Domain

Reply:
we have MD acronym all over this file. anyway, i changed to Memory Domain here.

Reply:
@bureddy This is something that we have to fix in the future. Maybe @tonycurtis can help.